.github/workflows/main-ci.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     - name: Setup Node.js
       uses: actions/setup-node@v4
       with:
-        node-version: 'latest'
+        node-version: '16.14'
```

Comment:
If there is a higher version available, we kindly recommend upgrading to the latest stable version for optimal performance and enhanced features.